python/packages/ai/teams/ai/augmentations/sequence_augmentation.py

```diff
@@ -41,13 +41,7 @@
                     "type": {"type": "string", "enum": ["DO", "SAY"]},
                     "action": {"type": "string"},
                     "parameters": {"type": "object"},
-                    "response": {
```

Comment:
@aacebo Corina and I were debugging and this is currently throwing an error in our test cases (its expecting type "string"- but we're still passing in the `Message` type which is an **object**, we may need to revert this code back

Reply:
<img width="1045" alt="image" src="https://github.com/microsoft/teams-ai/assets/14900841/3854a24e-0123-4a65-8068-94a877a41f39">


This is the error I'm getting. 

I also tried reverting back to this schema just now and surprisingly got the same exact error message.

Reply:
To clarify: we consoled the validation response and this is that response.